test/rules/no-unnecessary-constructor/test.ts.lint

```diff
@@ -0,0 +1,56 @@
+export class ExportedClass {
```

Comment:
needs tests with overload signatures, and parameter properties
also consider adding a test with an interface containing a construct signature

Reply:
+1